client/src/lib/stores/Interaction.ts

```diff
@@ -0,0 +1,4 @@
+import { writable } from 'svelte/store';
+/*The array here is an array of keys pressed by code (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code). 
+Limited to key presses relevant to functionality through checks where the store is modified.*/
+export const InteractionStore = writable<Array<String>>([]);
```

Comment:
I think this should be a Map<string, boolean>, this will allow for O(n) write and read speeds.

Also, just as a rule of thumb, primitives should always be used over their object implementations. Array<String> should be string[] if you need an array of strings.